src/services/webclient.ts

```diff
@@ -2172,11 +2172,16 @@ export class WebClientService {
             return future.reject('invalidResponse');
         }
 
-        // Store receivers
-        this.sortContacts(data.contact);
-        this.receivers.set(data);
-        this.registerInitializationStep(InitializationStep.Receivers);
-        future.resolve();
+        // Run delayed...
```

Comment:
Maybe add a comment, *why* this depends on the profile :slightly_smiling_face:

Reply:
"For technical reasons"? There's no real good rationale. It's just some assumption we make.

Reply:
Something like this:

> // Run delayed, because the receiver processing code depends on the profile being initialized.